engines/toltecs/toltecs.h

```diff
@@ -83,6 +83,16 @@ enum SysString {
 	kSysStrCount
 };
 
+enum TOLTECSAction {
+	kActionSkipDialog,
+	kActionOpenSaveMenu,
+	kActionOpenLoadMenu,
+	kActionSkipMovie,
+	kActionMenuOpen,
+	kActionSkipRide,
+	kActionNone,
```

Comment:
I recommend kActionNone at the top of the action enum as it seems an appropriate default value